ci/check_gucs_are_alphabetically_sorted.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -euo pipefail
+# shellcheck disable=SC1091
+source ci/ci_helpers.sh
+
+# extract citus gucs in the form of "citus.X"
+grep -o -E "(\.*\"citus.\w+\")," src/backend/distributed/shared_library_init.c > gucs.out
+sort -c gucs.out
```

Comment:
I do not like that after a `make reindent` we always dirty out git tree.

We can either delete the file, or add it to `.gitignore`

Reply:
That is a valid point, I think we should add it to gitignore, As an alternative we could also exclude this script from make reindent.

On a side note, I hope these scripts work on Mac, otherwise their `make reindent` will start failing.